terraform/hcl2json.go

```diff
@@ -41,46 +39,6 @@ func ParseModule(rawFiles map[string]interface{}) map[string]interface{} {
 	}
 }
 
-// extractInputVariables extracts the input variables values from the provided file
-var extractInputVariables = func(file File) (ValueMap, error) {
-	fileInputVariablesMap, diagnostics := extractInputVariablesFromFile(file)
-	if diagnostics.HasErrors() {
-		return ValueMap{}, createInvalidHCLError(diagnostics.Errs())
-	}
-
-	return fileInputVariablesMap, nil
-}
-
-// extractLocals extracts the input values from the provided file
-var extractLocals = func(file File, inputsMap ValueMap) (ValueMap, error) {
-	localsMap, diagnostics := extractLocalsFromFile(file, inputsMap)
-	if diagnostics.HasErrors() {
-		return ValueMap{}, createInvalidHCLError(diagnostics.Errs())
-	}
-
-	return localsMap, nil
-}
-
-// ParseHclToJson parses a provided HCL file to JSON and dereferences any known variables using the provided variables
-func ParseHclToJson(fileName string, fileContent string, variables ModuleVariables) (string, error) {
```

Comment:
Moved `ParseHclToJson` to `parser.go` because this file was getting confusing - couldn't tell which function was used externally and which internally